Runtime/Scripts/RGOverlayMenu.cs

```diff
@@ -173,6 +173,17 @@ public void StopBotInstance(long id)
             RGBotServerListener.GetInstance()?.TeardownClient((uint) id);
         }
 
+        /// <summary>
+        /// Tear down all currently active bots
+        /// </summary>
+        public void StopAllBots()
```

Comment:
Looks like this accidentally included work from your other branch

Reply:
Resolved. I committed a delete to avoid conflicts with other pulled versions. It _does_ muddle up the commit history so if you'd prefer to see a rollback + force push just let me know